news/2024/2024-03-19-changes-to-performance-points.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Changes to Performance Points
+date: 2024-03-19 18:00:00 +0000
+---
+
+Today peppy has graciously given (almost) everyone more performance points. Read on to find out why!
+
+![](/wiki/shared/news/banners/pp-sr-osu.jpg)
+
+As part of turning on performance points for osu!(lazer), there was a change in how much **bonus pp** would be awarded. In a [video by peppy and pishifat last month](https://www.youtube.com/watch?v=D7SZk8mU7Ww&t=40s), this change was mentioned to have been applied for the lazer client, and that it would be applied to stable at a later date. **Today is that day.**
+
+Apart from the [weighted sum of your top performances](/wiki/Performance_points#weightage-system), your total pp value also includes a bonus portion depending on how much you've played (in order to give new players a sense of progression). This bonus portion is 417 pp at its maximum and *used to be* achieved when you hit around 13,000 completed beatmap difficulties. That was changed to be around 1,500 difficulties.
+
+Here's a graph if you're a visual person (and here's the [interactive variant](https://www.desmos.com/calculator/ll2foxmgp3)):
+
+![](/wiki/shared/news/2024-03-19-changes-to-performance-points/performance-bonus-change.png)
+
+In short, you will find yourself having more bonus pp even if you haven't played 10,000+ maps.
+
+Also, since both osu!(lazer) and osu!(stable) are now using the same performance point processor, your rank graph will no longer flip-flop when you switch between clients. This means if your graph [looked](https://osu.ppy.sh/users/30070868) [something](https://osu.ppy.sh/users/5773079) [like](https://twitter.com/stompymymood/status/1769833222450405756) [this](https://twitter.com/Vid_osu/status/1770132123141251427)...
```

Comment:
```suggestion
Also, since both osu!(lazer) and osu!(stable) are now using the same performance point processor, scores set on osu!(lazer) will always be included in your PP value – even when playing on osu!(stable).

Your rank graph will also no longer flip-flop when you switch between clients. This means if your graph [looked](https://osu.ppy.sh/users/30070868) [something](https://osu.ppy.sh/users/5773079) [like](https://twitter.com/stompymymood/status/1769833222450405756) [this](https://twitter.com/Vid_osu/status/1770132123141251427)...
```